lib/netext/httpext/request.go

```diff
@@ -217,7 +218,12 @@ func MakeRequest(ctx context.Context, preq *ParsedHTTPRequest) (*Response, error
 
 	// Only set the name system tag if the user didn't explicitly set it beforehand
 	if _, ok := tags["name"]; !ok && state.Options.SystemTags["name"] {
-		tags["name"] = preq.URL.Name
+		var name = preq.URL.Name
+		if state.Options.AnonymizeQueryStringValues.Bool {
+			var qsValueRegex = regexp.MustCompile("=[^&]*")
```

Comment:
You should put this regex in a global var, otherwise, we waste time to re-compile it every time makes request.

Or it's better to don't use regex anyway, a simple for loop or using strings.Index.

Reply:
I extracted it into a global var.